randomwallpaper@iflow.space/elements.js

```diff
@@ -129,6 +131,15 @@ var HistoryElement = GObject.registerClass({
 		});
 		this.menu.addMenuItem(this.copyToFavorites);
 
+		// Static URLs can't block images
+		if (historyEntry.adapter.type !== 5) {
```

Comment:
I moved that enum from the schema into the code now. I don't see the benefit having it in the schema. The string representation is hardcoded and accessing happens through an integer anyway while it's not easily possible to compare, switch or iterate through them without duplicating the enum inside the code.

This changes the key type from string to int - so if there are already some values in that schema/path/key combination from testing it might fail after that change.

[`15ac148` (#154)](https://github.com/ifl0w/RandomWallpaperGnome3/pull/154/commits/15ac1489ddbf74a1a87200a243c180410a461a3a)

Reply:
I think this is a good change and makes the code more readable. And you are right, there isn't really a benefit of having an enum in the scheme (except for preventing incompatible values).